pipelines.adoc

```diff
@@ -1,266 +1,189 @@
+. Log in to your cluster. For example for OKD,
 
-....
+```
 oc login <master node IP>:8443
-....
+```
 
-==== Clone the kabanero pipelines repo
+. Clone the kabanero pipelines repo
 
-....
+```
 git clone https://github.com/kabanero-io/kabanero-pipelines
-....
+```
 
-==== Run the script with the appropriate parameters
+. Run the following script with the appropriate parameters
 
-....
+```
 cd ./pipelines/incubator/manual-pipeline-runs/
 
 ./manual-pipeline-run-script.sh -r [git_repo of the Appsody project] -i [docker registery path of the image to be created] -c [collections name of which pipeline to be run]"
-....
+```
 
 For example:
 
-....
+```
 ./manual-pipeline-run-script.sh -r https://github.com/mygitid/appsody-test-project -i index.docker.io/mydockeid/my-java-microprofile-image -c java-microprofile"
-....
+```
 
-=== Manual pipeline execution via command line
+=== Manual pipeline execution from the command line
 
-Login to your cluster. For example for OKD,
+. Login to your cluster. For example for OKD,
 
-....
+```
 oc login <master node IP>:8443
-....
+```
 
-==== Clone the kabanero pipelines repo
+. Clone the kabanero pipelines repo
 
-....
+```
 git clone https://github.com/kabanero-io/kabanero-pipelines
 cd kabanero-pipelines
-....
+```
 
-==== Create PipelineResources
+. Create Pipeline resources
 
-Update the pipeline-resources.yaml with github & docker repo info to
-create the PipelineResources. Sample pipeline-resources.yaml files are
-provided for each featured collection under the manual-pipeline-runs
-dir. Update the docker-image URL. You can use the sample github repo
-provided or update it to point to your github repo.
+Update the `pipeline-resources.yaml` file with your GitHub & Docker repo information to create the `PipelineResources`. Sample `pipeline-resources.yaml` files are provided for each featured collection in the `manual-pipeline-runs` directory. Update the docker-image URL. You can use the sample GitHub rep or update it to point to your own GitHub repo.
 
-After updating the file, apply it
+. After you update the file, apply it.
 
-....
+```
 oc apply -f <collection-name>-pipeline-resources.yaml
-....
+```
 
-==== Activate the tasks & pipelines
+==== Activate tasks & pipelines
 
-The installations will activate the featured collections should have
-activated the tasks and pipelines already. If you are creating a new
-task or pipeline, activate them manually
+The installations that activate the featured collections also activate the tasks and pipelines. If you are creating a new task or pipeline, activate it manually, as shown in the following example.
 
-....
+```
 oc apply -f <task.yaml>
 oc apply -f <pipeline.yaml>
-....
+```
 
 ==== Run the pipeline
 
-Sample PipelineRun files are provided under
-./pipelines/manual-pipeline-runs. Locate the appropriate pipeline-run
-file and execute it.
+Sample PipelineRun files are provided under `./pipelines/manual-pipeline-runs`. Locate the appropriate pipeline-run
```

Comment:
We made a recent change here.  So need to update the doc here to say:

A sample pipelinerun yaml is provided under `./pipelines/manual-pipeline-runs`.  Rename the template file to <collection-name>-pipeline-run.yaml for example and update the file to replace "collection-name" with the real collection name.